AutoPong/.vscode/launch.json

```diff
@@ -0,0 +1,34 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Android Debug",
+            "type": "coreclr",
```

Comment:
should this be `mono` not `coreclr`? does `coreclr` even work?

Reply:
@dellis1972 Yes `coreclr` does work. The only blocker now with Android is that it correctly uploads to the running simulator, but then prompts for something to attach (could this be because of `coreclr` instead of `mono`):
![Screenshot_1712331114](https://github.com/MonoGame/MonoGame.Samples/assets/271363/0967dbfd-ff3f-41d8-98c0-a92c43b5331d)

VSCode says:

![image](https://github.com/MonoGame/MonoGame.Samples/assets/271363/822089ea-679d-4131-9826-488605c00afe)

Reply:
Scrap that, I'll add the mono dependency, as it makes this SO much easier :D